tests/contract/test_resource_client.py

```diff
@@ -80,6 +80,43 @@
     "primaryIdentifier": ["/properties/c", "/properties/d"],
 }
 
+SCHEMA_WITH_TRANSFORM = {
```

Comment:
Why is this needed?

Reply:
This is not part of my change, but got pulled into the diff due to a merge when rebasing.

Reply:
Removed the extraneous lines from the commit, sorry for the noise.